cmd/erasure-multipart.go

```diff
@@ -289,9 +289,10 @@ func (er erasureObjects) newMultipartUpload(ctx context.Context, bucket string,
 		parityDrives = er.defaultParityCount
 	}
 
-	ecOrg := parityDrives
+	parityOrig := parityDrives
 	for _, disk := range onlineDisks {
 		if parityDrives >= len(onlineDisks)/2 {
+			parityDrives = len(onlineDisks) / 2
```

Comment:
@harshavardhana Are you assuming that `globalStorageClass.GetParityForSC` can return values `> len(onlineDisks) / 2`, otherwise this does nothing?

Reply:
It is a possibility of bug to avoid.